### ISSUE TITLE
[Bug]: Intercepting nodejs in a terminal breaks Alpine docker builds

### ISSUE THREAD

**Issue by Caerbannog** [httptoolkit/httptoolkit#332]
### Repro steps

Reproduction:
1. Open the desktop app
2. Click "Fresh Terminal" or "Existing Terminal"
3. Create a `Dockerfile` based on Alpine Linux that uses nodejs (see below)
4. Run `docker build`
5. An error message is displayed: `env: can't execute 'bash': No such file or directory`

Example Dockerfile that breaks on line 2:
```dockerfile
FROM node:16.13.0-alpine
RUN npm --version
CMD npm run start
```

### How often does this bug happen?

Often

### The desktop OS you're using

Ubuntu 22.04

### Error screenshot

![image](https://user-images.githubusercontent.com/242172/191933950-5deb5f08-d0e2-401c-ad9a-0b60514c121c.png)

**Comment by pimterry** [comment]
Hi @Caerbannog, thanks for reporting this! I appreciate the clear repro steps, that's great. I have actually seen this elsewhere, I believe I have a working fix ready to go, which will be included in the next release. I'll keep this ticket open until that's live though, it should be within the next week or so.

In the meantime, you can work around this by manually installing bash with `apk add --no-cache bash`.

**Comment by pimterry** [comment]
This has now been released in server v1.10.3. Your HTTP Toolkit will update in the background automatically next time it starts, and the following time everything should work correctly. Thanks for reporting this @Caerbannog! I'll close this now, but feel free to reply here or open a new issue if you're still having problems.
